cmd/plural/crypto.go

```diff
@@ -121,10 +121,13 @@ func cryptoCommands() []cli.Command {
 	}
 }
 
-func handleEncrypt(c *cli.Context) error {
+func handleEncrypt(*cli.Context) error {
```

Comment:
when I run the linter locally I'm not getting this change for some reason.

Reply:
Nevermind, issue with my setup.

Reply:
@zreigz Actually it does seem like this change isn't picked up when I run it locally. What command are you using to run `golangci-lint`?

Reply:
I had this when I was using go 1.17, now I changed to 1.18 and now it's not valid. I will revert